src/shared/lib/StoreUtils.ts

```diff
@@ -98,6 +98,20 @@ export async function fetchPdbAlignmentData(ensemblId: string,
     }
 }
 
+export function fetchPtmData(ensemblId: string,
+                             client:GenomeNexusAPI = genomeNexusClient): Promise<PostTranslationalModification[]>
+{
+    if (ensemblId)
```

Comment:
is there a reason there wouldn't be an ensemble id?

Reply:
This is to ignore empty string, but I guess I am being too cautious. This would be more clear:
```suggestion
    if (ensemblId.length > 0)
```